src/Text.js

```diff
@@ -150,17 +150,29 @@ const Typography = (props) => {
   const textStyles = StyleSheet.flatten([
     {
       fontWeight: WEIGHTS.regular,
-      fontSize: SIZES.font,
+      fontSize: usePixelRatio(SIZES.font / 2),
```

Comment:
https://stackoverflow.com/a/42971004

I was trying to implement this. I divided all sizes by 2 because i found that it remains similar to how it looked before implementing it. Not dividing by 2 makes the font size huge. I'll move the division part to the usePixelRatio hook itself and call it the `scale` factor. Having a default value of 0.5 but will be configurable if passed as a function parameter

```
usePixelRatio(pixel, scale = 0.5)
```